qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -683,15 +833,12 @@ def reservations(
     ) -> List[BackendReservation]:
         return []
 
-    def run(
+    def run(    # type: ignore[override]
             self,
-            qobj: Union[QasmQobj, PulseQobj],
-            job_name: Optional[str] = None,
-            job_share_level: Optional[str] = None,
-            job_tags: Optional[List[str]] = None,
-            validate_qobj: bool = False
+            **kwargs: Dict
```

Comment:
Won't this fail for cases where qobj was passed in before. It's probably not a big deal except that `run(qobj)` will raise a TypeError now while it would raise an `IBMQBackendError` before